rules/false_positives/tensorflow_model_analysis.yara

```diff
@@ -0,0 +1,14 @@
+rule tfjs_predict_extractor_util : override {
+  meta:
+    description = "tfjs_predict_extractor_util.py"
+    original_severity = "critical"
+    py_dropper_chmod = "high"
```

Comment:
medium

Reply:
Addressed in [`e716ba1` (#502)](https://github.com/chainguard-dev/malcontent/pull/502/commits/e716ba11a75e6cbd2e99b0f54f0d82f3deb46bef).